package.json

```diff
@@ -34,27 +39,28 @@
     "async": "^2.6.1",
     "browserify-aes": "^1.2.0",
     "bs58": "^4.0.1",
+    "iso-random-stream": "^1.1.0",
     "keypair": "^1.0.1",
-    "libp2p-crypto-secp256k1": "~0.2.2",
+    "libp2p-crypto-secp256k1": "libp2p/js-libp2p-crypto-secp256k1#fix/bundle-size",
     "multihashing-async": "~0.5.1",
     "node-forge": "~0.7.6",
     "pem-jwk": "^1.5.1",
     "protons": "^1.0.1",
     "rsa-pem-to-jwk": "^1.1.3",
     "tweetnacl": "^1.0.0",
-    "ursa-optional": "~0.9.9",
-    "webcrypto-shim": "github:dignifiedquire/webcrypto-shim#master"
```

Comment:
are you sure you can just drop these shims? I don’t think the state of webcrypto has improved much since this was written

Reply:
for the current support/target browserlist yes we can